lang/en.json

```diff
@@ -566,7 +566,13 @@
         "VerifyAccount": "Verifying Account",
         "ValidateSslCertificate": "Validate SSL Certificate",
         "VerifyLoginData": "Make sure your username and password combination is correct.",
-        "YouAreOffline": "Sorry, you are currently offline"
+        "YouAreOffline": "Sorry, you are currently offline",
+        "ExceptionNoViewAccess": "Please check your username and password and make sure you have %s access for at least one website.",
+        "Mobile_HowtoExitAndroid": "Please click BACK again to exit",
+        "PiwikMarketplace": "Matomo Marketplace",
```

Comment:
Maybe we should directly change that key to `MatomoMarketplace` ?

Reply:
sure, will change it